cinema/serializers.py

```diff
@@ -111,10 +136,18 @@ class MovieSessionDetailSerializer(MovieSessionSerializer):
     taken_places = TicketSeatsSerializer(
         source="tickets", many=True, read_only=True
     )
+    movie_image = MovieImageSerializer(many=False, read_only=True)
```

Comment:
you already have image inside MovieListSerializer - so you don't need to include it twice